consensus/src/chained_bft/epoch_manager.rs

```diff
@@ -328,12 +330,13 @@ impl<T: Payload> EpochManager<T> {
     // event processor at startup. If we need to sync up with peers for blocks to construct
     // a valid block store, which is required to construct an event processor, we will take
     // care of the sync up here.
-    async fn start_sync_processor(&mut self, ledger_recovery_data: LedgerRecoveryData) {
+    fn start_sync_processor(&mut self, ledger_recovery_data: LedgerRecoveryData) {
         let epoch_info = ledger_recovery_data.epoch_info();
-        self.network_sender
-            .update_eligible_nodes(ledger_recovery_data.validator_keys())
-            .await
-            .expect("Unable to update network's eligible peers");
+        block_on(
```

Comment:
can we please make this always futures::block_on? not import the function itself?

Reply:
is this a guideline somewhere?

Reply:
I think it makes it more readable. Because the Tokio runtime also has a block_on function. I read in the Rust book that we should never import functions directly.

Reply:
But I cannot find it.

Reply:
Runtime block_on is associated function like `runtime.block_on`, the full name is just too long to type `futures::executor::block_on`

Reply:
Since you should have very few of these calls in general, I think it's ok to have the name be long.

Reply:
executor::block_on is ok, but we have some really crazy long function names and variables :P